backend/typescript/models/camper.model.ts

```diff
@@ -6,10 +6,11 @@ export interface Camper extends Document {
   firstName: string;
   lastName: string;
   age: number;
-  contactName: string;
+  parentName: string;
```

Comment:
showing changes to old versions of this file, can we discard all modifications to this file?